frontend/src/pages/AllRunsList.tsx

```diff
@@ -77,11 +78,11 @@ class AllRunsList extends Page<{}, AllRunsListState> {
   private _selectionChanged(selectedIds: string[]): void {
     const toolbarActions = [...this.props.toolbarProps.actions];
     // Compare runs button
-    toolbarActions[1].disabled = selectedIds.length <= 1 || selectedIds.length > 10;
```

Comment:
The design of this toolbarActions array is a source of potential problems, I kind of regret doing it this way. I'd recommend adding a TODO to refactor it into an object with tighter type semantics.

Reply:
Agreed. Added a TODO